app/extras/queries/visible_motions.rb

```diff
@@ -7,7 +7,7 @@ def initialize(user: nil, groups: nil, group_ids: nil)
       group_ids = Array(groups).map(&:id)
     end
 
-    @relation = Motion.joins(:discussion => :group).merge(Group.published).preload(:discussion)
+    @relation = Motion.joins(discussion: :group).where('groups.archived_at IS NULL').preload(:discussion)
```

Comment:
Does this seem ok @robguthrie ? We were getting errors with the default include because we were merging Motion relations with Group relations.